### PR TITLE
Add initial labels

### PR DESCRIPTION
**Pull request by BWBellairs**
Ref: #12 
Signed-off-by: BWBellairs <BWBellairs@outlook.com>

### CODE REVIEW

File: bot/index.js [hunk 1]
```diff
@@ -130,9 +131,10 @@ module.exports = app => {
       issueNumber = payload.issue.number,
       issueText = `${payload.issue.title.substring(0, 150)}${payload.issue.title.length > 150 ? '...' : ''}`;
 
+    let label = (labels)[payload.label.name] || payload.label.name
```
Comment (wolfy1339) [inline bot/index.js]: The parenthesis aren't needed here

File: bot/index.js [hunk 2]
```diff
@@ -130,9 +131,10 @@ module.exports = app => {
...
     await shortenUrl(payload.issue.html_url, url => {
-      app.irc.privmsg(`${att} \x0F| ${user} ${action}\x0F `
-          + `issue #${issueNumber} with label ${color}${payload.label.name}\x0F (${issueText}) - ${url}`);
+      app.irc.privmsg(`${att} \x0F| ${user} ${color + action}\x0F `
```
Comment (wolfy1339) [inline bot/index.js]: Please don't use the `+` selector like that.

Comment (BWBellairs) [reply bot/index.js]: That's a perfectly valid use as far as I'm concerned. Unless you want me to format it like ${color}...

Comment (wolfy1339) [reply bot/index.js]: Yes, that's what i mean

**Pull request merged**